lib/packs/rails/integrations/rails.rb

```diff
@@ -27,6 +29,10 @@ def create_engines
         def inject_paths
           Packs.all.reject(&:is_gem?).each do |pack|
             Packs::Rails.config.paths.each do |path|
+              # Prior to Rails 6.1, the "config/routes" app path is nil and was not added until drawable routes feature was implemented
+              # https://github.com/rails/rails/pull/37892/files#diff-a785e41df3f87063a8fcffcac726856a25d8eae6d1f9bca2b36989fe88613f8eR62
+              next if path == CONFIG_ROUTES_PATH && ::Rails.gem_version < Gem::Version.new("6.1")
```

Comment:
It'd be worth extracting and caching the result of `::Rails.gem_version < Gem::Version.new("6.1")`. It's not that it's particularly slow, but extra object allocations can add up during app boot.

I could see it being called like `pre_rails_61?` or something more semantic like `config_routes_app_path_nil?`

Reply:
Updated. Please feel free to update the code directly as well if you have a specific approach in mind.